spark/src/test/java/com/netflix/iceberg/spark/source/TestFilteredScan.java

```diff
@@ -429,7 +429,9 @@ private File buildPartitionedTable(String desc, PartitionSpec spec, String udf,
     Table byId = TABLES.create(SCHEMA, spec, location.toString());
 
     // do not combine splits because the tests expect a split per partition
-    byId.updateProperties().set("read.split.target-size", "1").commit();
+    //TODO: this is commented out since the current patch will create
+    // too many small scan tasks
+    // byId.updateProperties().set("read.split.target-size", "1").commit();
```

Comment:
This was set to avoid combining splits. Instead of setting it to 1, it works to set it to some value larger than the file size to avoid splitting. But the value can't be too large, or else splits get combined. Looks like 2048 works so that no files are split or combined.

Reply:
Thankyou :)